app/my-dashboard/_features/rewards-table/_components/filter-data/filter-data.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "@/design-system/atoms/button/variants/button-default";
+
+import { useFilterData } from "@/shared/features/filters/_contexts/filter-data/filter-data.context";
+import { ContributorProjectFilter } from "@/shared/features/filters/contributor-project-filter/contributor-project-filter";
+import { CurrencyFilter } from "@/shared/features/filters/currency-filter/currency-filter";
+import { SidePanelBody } from "@/shared/features/side-panels/side-panel-body/side-panel-body";
+import { SidePanelFooter } from "@/shared/features/side-panels/side-panel-footer/side-panel-footer";
+import { SidePanelHeader } from "@/shared/features/side-panels/side-panel-header/side-panel-header";
+import { useSidePanel } from "@/shared/features/side-panels/side-panel/side-panel";
+import { Translate } from "@/shared/translation/components/translate/translate";
+
+import { RewardsTableFilters } from "../../rewards-table";
+import { useProjectRewardsFilterDataSidePanel } from "./filter-data.hooks";
+
+export function FilterData() {
+  const { name } = useProjectRewardsFilterDataSidePanel();
+  const { Panel } = useSidePanel({ name });
+  const { filters, setFilters, saveFilters, resetFilters } = useFilterData<RewardsTableFilters>();
+
+  return (
+    <Panel>
+      <SidePanelHeader
+        title={{
+          children: <Translate token={"manageProjects:detail.filters.titles.rewards"} />,
```

Comment:
You're using the manage projects translations

Reply:
filter are not done yet but a can put the good translation